documentation/bolt_known_issues.md

```diff
@@ -1,5 +1,33 @@
 # Known issues
 
+## `facts` task fails on Windows targets with Facter 3 installed
+
+When running the `facts` task on a Windows target that has Facter 3 installed,
```

Comment:
Does this happen on all Windows versions consistently?

Reply:
I tried with 2012 r2, 2016, and 2019 images and it showed on all 3.